app/core/models/ocelot/stanzas/CalculationStanza.scala

```diff
@@ -54,63 +58,73 @@ sealed trait Operation {
   val right: String
   val label: String
 
-  def eval(labels: Labels) : Labels
+  def eval(labels: Labels): Labels
 
-  def value(arg: String, labels: Labels): String = labelReference(arg).fold(arg){ref => labels.value(ref).getOrElse("")}
+  def value(arg: String, labels: Labels): String = labelReference(arg).fold(arg) { ref =>
+    labels.value(ref).getOrElse("")
+  }
 
   def toPlainString(value: BigDecimal): String = value.bigDecimal.toPlainString
 
-  def addString(s1: String, s2: String) : Option[String] = Option(s1 + s2)
+  def addString(s1: String, s2: String): Option[String] = Option(s1 + s2)
 
-  def unsupportedOperation(s1: String, s2: String) : Option[String] = {
+  def unsupportedOperation(arg1: Any, arg2: Any): Option[String] = {
 
     logger.error("Unsupported " + getOperation.toLowerCase + " calculation stanza operation defined in guidance")
 
     None
   }
 
-  private def getOperation : String = this.getClass.getSimpleName.replace("Operation", "")
+  private def getOperation: String = this.getClass.getSimpleName.replace("Operation", "")
```

Comment:
as toPlainString

Reply:
This method will be removed after change to curried function.